libraries/vmmanager/vm_backend.cpp

```diff
@@ -0,0 +1,54 @@
+
+#include <koinos/vmmanager/exceptions.hpp>
+#include <koinos/vmmanager/vm_backend.hpp>
+
+#include <koinos/vmmanager/eos/eos_vm_backend.hpp>
+#include <koinos/vmmanager/fizzy/fizzy_vm_backend.hpp>
+
+#include <cstdlib>
+
+namespace koinos::vmmanager {
+
+vm_backend::vm_backend() {}
+vm_backend::~vm_backend() {}
+
+std::vector< std::shared_ptr< vm_backend > > get_vm_backends()
+{
+   std::vector< std::shared_ptr< vm_backend > > result;
+
+   result.push_back( std::make_shared< vmmanager::eos::eos_vm_backend >() );
+   result.push_back( std::make_shared< vmmanager::fizzy::fizzy_vm_backend >() );
+
+   return result;
+}
+
+std::string default_vm_backend = "fizzy";
+
+std::string get_default_vm_backend_name()
+{
+   const char* env_backend = std::getenv("KOINOS_VM_BACKEND");
```

Comment:
We should not use environment variables for this, everything needs to happen inside the `config.yml` file but seeing as though we want to remove EOS from the codebase I think it isn't necessary right now.

Return `"fizzy"` hardcoded.

Reply:
I agree there should be a way to do it in the config file.  However we may also want to have the capability to run unit tests under both VM's.

I think this should be part of #459 instead of the current issue.